### DISCUSSION TITLE
replicating Bulk modulus calculated from structures in matterverse

### DISCUSSION THREAD

**Discussion by KarimElgammal** [materialsvirtuallab/matcalc#9] [Q&A] [answered]
Hi,

thx for matcalc and the classes that calculate the EOS for example

I was calculating the properties for https://matterverse.ai/details/mp-6930 using matcalc but it produces bulk modulus in ''bulk_modulus_bm'': 84.61 GPa for M3GNet and ''bulk_modulus_bm'': 70.46 GPa for CHGNet that is quite different from the one reported in mattervese link which is 33.38 GPa. Given that experimental value is 38 or 40 GPa as reported in (J Mcskimin, P Andreatch, RN Thurston J Appl Phys, 1987, M Grimsditch, A Polian, V Brazhkin, D Balitski˘ı J Appl Phys, 1998)

I believe it is a very nice way already calculating the EOS and Bulk modulus in matcalc, but not sure why it has big difference with matervese or experimental values. Can you help me in that

**Comment by JiQi535** [comment, marked as answer]
@KarimElgammal Hi Karim, I would like to point out that the methods to calculate bulk modulus are different in matterverse and in matcalc. 

In matcalc, we use universal **interatomic potentials** (M3GNet, CHGNet) to simulate the strain-stress relation and get the elastic tensor and bulk modulus. While in matterverse, the bulk modulus and shear modulus are predicted with **M3GNet property models** trained with matbench data (labeled in matterverse webpage). These property models are trained and benchmarked in the [original M3GNet work](http://dx.doi.org/10.1038/s43588-022-00349-3), and their parameters can be found in figshare ([see discussion here](https://github.com/materialsvirtuallab/m3gnet/blob/main/pretrained/matbench_properties.md)).

**Reply by KarimElgammal** [reply]
thx allot!
